frontend/modules/teamspace/teamspace.redux.ts

```diff
@@ -17,7 +17,7 @@
 
 import { createActions, createReducer } from 'reduxsauce';
 
-const getAvatartUrl = (username) => `/api/${username}/avatar?${Date.now()}`;
+const getAvatartUrl = (username) => `${ClientConfig.apiUrls.all[0]}/${username}/avatar?${Date.now()}`;
```

Comment:
I'm not sure if you've done the same else where, but just noticed it. 

This shouldn't be using the first API URL always or it defeats the point of it. All the angular stuff uses the apiURL function get the URL generated from a round robin algorithm. The reason being is that we want to bypass the 15 http request per domain limit on the browser. So we need to make sure we're not just using the first API domain everytime.

If you're doing this on other parts of the React migration please change it! :)

Reply:
Thinking about it, shoudln't this be going through the api service instead of trying to construct the url directly?

Reply:
> If you're doing this on other parts of the React migration please change it! :)

Sure! I'll change it!

> Thinking about it, shoudln't this be going through the api service instead of trying to construct the url directly?

Yes, you have absolutely right.